.ci/Generation/SchemaGeneration/SchemaGeneration/Generate.cs

```diff
@@ -0,0 +1,70 @@
+﻿using BH.Engine.JsonSchema;
+using BH.Engine.Base;
+using BH.oM.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using BH.oM.JsonSchema;
+
+namespace SchemaGeneration
+{
+    public static class Generate
+    {
+        public static void GenerateAssembly(Assembly assembly, ConvertConfig config)
+        {
+            var types = assembly.GetTypes();
+
+            //Clean the folder before generating new schemas        
+            if (Directory.Exists(System.IO.Path.Combine(Path, assembly.GetName().Name)))
+            {
+                Directory.Delete(System.IO.Path.Combine(Path, assembly.GetName().Name), true);
+            }
```

Comment:
I'm not a big fan on how this is done. You're relying on your knowledge of how `type.GetPath()` works. But if anything changes there, this will not work as intended anymore.